### DISCUSSION TITLE
Is there any way to get value in function?

### DISCUSSION THREAD

**Discussion by hasunpark** [facebookexperimental/Recoil#1436] [Q&A] [answered]
Hi! First of all thanks for working this great library

I want to know can i get value inside function without subsribe?

Like using useRef.current

**Comment by drarmstr** [comment, marked as answer]
Yes, this is possible with [`useRecoilCallback()`](https://recoiljs.org/docs/api-reference/core/useRecoilCallback).

**Reply by hasunpark** [reply]
Thank you! i saw all document before, but useRecoilCallback was hook so i thought i can not use it in functions. But i think i miss understand how to use it!! Thanks for your support 💯 

i will try to use it later project
